src/lib/containers/IconSvg.tsx

```diff
@@ -233,6 +234,8 @@ const getIcon = (options: IconSvgOptions) => {
       return <ReplyTwoTone style={muiSvgStyle}></ReplyTwoTone>
     case 'chatTwoTone':
       return <ChatTwoTone style={muiSvgStyle}></ChatTwoTone>
+    case 'accessManagement':
+      return <AccessManagement></AccessManagement>
```

Comment:
Add support for fill color and customSvgStyle like many other custom svg icons in here?